mantidproject/mantid
# PANTHER IDF fix

**Description of work.**
Corrects the IDF of PANTHER following the calibration measurements during the hot commissioning.
Confirmed by the instrument scientists.
The PR of the generator can be found [here](https://github.com/mantidproject/mantidgeometry/pull/174)

**To test:**
1) `rm -rf ~/.mantid/instrument`
2) Run LoadEmptyInstrument with PANTHER as the name.
3) Open Instrument Viewer, go to pick tab.
4) Mouse hover on 80th pixel of any of the tubes. Make sure that for those `y=0`, `r=2.5`.





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
